lib/npg_pipeline/base.pm

```diff
@@ -240,8 +257,13 @@ Boolean decision to force on P4 pipeline usage
 has q{force_p4}  => (
   isa           => q{Bool},
   is            => q{ro},
-  documentation => q{Boolean decision to force on P4 pipeline usage},
+  lazy_build    => 1,
+  documentation => q{Boolean decision to force on P4 pipeline usage, default true iff GCLP},
```

Comment:
Ummm: I meant [if and only if](https://en.wikipedia.org/wiki/If_and_only_if). I agree "if" would suffice for this description though.